### ISSUE TITLE
Remove tabindex from A11y toggleEnabled()

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-core#220]
### Subject of the issue/enhancement/features
When using a11y.toggleEnabled, a tabindex attribute of -1 is added.  This results in the button not being focusable via keyboard navigation.

This function also adds aria-disabled="false".  If we remove tabindex="-1" but keep aria-disabled="false", the user would be able to tab to the element and understand that it's disabled.  This may be a better user experience than not allowing the user to tab to the element altogether.

### Your environment
* Framework 5.x

### Steps to reproduce
Add a Hotgraphic component to a course.  Click on an item to open the Notify popup.  Note that the paging buttons call a11y.toggleAccessibleEnabled to enable or disable the buttons.  That function then calls a11y.toggleEnabled.

### Expected behaviour
Disabled buttons should be visible, focusable and marked as disabled.

### Actual behaviour
Disabled buttons are hidden and not focusable.

### Considerations for other plugins
As the disabled buttons would now be focusable, some plugin styles may need to be updated.  For instance, the Hotgraphic disabled paging buttons should probably be made visible (i.e. remove `visibility: hidden`) and styled to resemble a disabled button state.

**Comment by oliverfoster** [comment]
love it
